doc/man3/X509_STORE_CTX_get_error.pod

```diff
@@ -103,6 +103,10 @@ The operation was successful.
 
 Unspecified error; should not happen.
 
+=item B<X509_V_ERR_INVALID_CERTIFICATE_FORMAT: certificate representation has invalid format>
```

Comment:
See proposed simplification of the error message.

Reply:
Changed as well.